EPRN.Portal/Areas/Exporter/Controllers/PRNSController.cs

```diff
@@ -247,6 +247,42 @@ public async Task<IActionResult> DecemberWaste(DecemberWasteViewModel decemberWa
                                     new { decemberWaste.Id });
         }
 
+        [HttpGet]
+        public async Task<IActionResult> ActionPrn(int? id)
```

Comment:
ActionPrn is not very meangingful - if you want to keep that as an action name then use the Action attribute, and then rename the method to something a little bit more meaningful.

Reply:
Done.